annotations/models.py

```diff
@@ -641,7 +641,7 @@ def dateRepresentation(self):
             _rep += '-' + str(self.day).zfill(2)
         return _rep
 
-    def __unicode__(self):
+    def __str__(self):
```

Comment:
Why did you change these from unicode to str?

Reply:
Its the change from Python 2 to Python 3. 
Changelog - https://docs.djangoproject.com/en/1.11/topics/python3/#str-and-unicode-methods

Reply:
Got it, it interesting that is didn't automatically change all __unicode__ methods to __str__ methods when we ran 2to3.